controllers/policy/targettracking/policy.go

```diff
@@ -49,6 +64,38 @@ func processBestContrast(base float64, trackingTargetVal float64, current float6
 	return int64(math.Ceil(base * rate))
 }
 
+func checkScheduledValue(scheduled []rebalancev1.Scheduled, v int64, nowTime time.Time) (int64, error) {
+	var values []int
+	values = append(values, int(v))
+	for _, s := range scheduled {
+		startTime, err := parseTime(s.StartTime, nowTime)
+		if err != nil {
+			return 0, err
+		}
+		endTime, _ := parseTime(s.EndTime, nowTime)
+		if err != err {
+			return 0, err
+		}
+		if (nowTime.Equal(startTime) || nowTime.After(startTime)) && nowTime.Before(endTime) {
+			values = append(values, int(s.Value))
+		}
+	}
+	return int64(funk.MaxInt(values)), nil
+}
+
+func parseTime(t string, n time.Time) (time.Time, error) {
+	ts := strings.Split(t, ":")
```

Comment:
Why not use time.Parse?

Reply:
With the content you suggested, the afternoon time in the format like "19:00" will not be parsed.

```go
func main() {
	t := "19:00"
	tt, _ := time.Parse("3:04", t)
	fmt.Println(tt) // 0001-01-01 00:00:00 +0000 UTC
}
```